src/python/python/ad/integrators/common.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, props = mi.Properties()):
         # Warn about potential bias due to shapes entering/leaving the frame
         self.sample_border_warning = True
 
-    def aovs(self):
+    def aovs_names(self):
```

Comment:
The method in C++ is actually called `aov_names` and we already have bindings for it, so we don't even need to re-define it here I believe.

Reply:
Ouach nice catch!